src/CardInfo.js

```diff
@@ -3,11 +3,73 @@ import React from 'react';
 import './App.css';
 
 export class CardInfo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isEditing: false,
+            oldDescription: props.description,
```

Comment:
I don't think you need it

Reply:
> I don't think you need it


the state.oldDescription field is a must, the description will be stored in it, until editing mode is enabled. state.description will be at the moment the user enters textarea